.github/workflows/release-homebrew.yaml

```diff
@@ -85,8 +88,34 @@ jobs:
           brew bump-cask-pr --write-only -f --version=$FINCH_VERSION finch
       - name: Silently install
         run: |
+          export HOMEBREW_NO_INSTALL_FROM_API=1
           cd $(brew --repo homebrew/cask)
           brew install --cask ./Casks/finch.rb
+      - name: Build project
+        run: |
+          brew install go lz4 automake autoconf libtool
+          export PATH="/opt/homebrew/opt/libtool/libexec/gnubin:$PATH"
+          which libtool
+          make
+      - name: Multiple instances of Finch test
+        run: |
+          pwd
+          # start two Finch VM instances
+          ./_output/bin/finch vm init
+          finch vm init
+          # start a container in each VM instance
+          ./_output/bin/finch pull alpine
+          finch pull alpine
+          ./_output/bin/finch run -d --name test-ctr1 alpine
```

Comment:
`./_output/bin/finch run -d --name test-ctr1 alpine`
`finch run -d --name test-ctr2 alpine`
Dont we need to verify whether the container run properly or not using ps cmd? what if it failed with an error?

Reply:
Removed the `-d` flag and ddded check whether the container exists in `ps -a` command output.

Reply:
Added